lib/locales/el.ts

```diff
@@ -0,0 +1,501 @@
+    john_mary_vianney_priest: 'Αγίου Ιωάννη Μαρία Vianney, πρεσβυτέρου',
+    john_of_capistrano_priest: 'Αγίου Ιωάννη de Capestrano, πρεσβυτέρου',
+    john_of_god_duarte_cidade_religious: 'Αγίου Ιωάννη του Θεού, μοναχού',
+    john_of_kanty_priest: 'Αγίου Ιωάννη de Kety, πρεσβυτέρου',
+    john_of_the_cross_priest: 'Αγίου Ιωάννη του Σταυρού, πρεσβυτέρου και διδασκάλου της Εκκλησιάς',
+    john_paul_ii_pope: 'Αγίου Ιωάννη Παύλου Βʹ, πάπα',
+    john_xxiii_pope: 'Αγίου Ιωάννη ΚΓ΄, πάπα',
+    josaphat_kuntsevych_bishop: 'Αγίου Ιωσαφάτ, επισκόπου Και μάρτυρος',
+    joseph_of_calasanz_priest: 'Αγίου Ιωσήφ de Calasanz, πρεσβυτέρου',
+    joseph_spouse_of_mary: 'Αγίου Ιωσήφ, Μνήστορος της Αειπαρθένου Μαρίας',
+    joseph_the_worker: 'Αγίου Ιωσήφ, εργαζομένου',
+    josephine_bakhita_virgin: 'Αγίας Ιωσηφίνας Bakhita, παρθένου',
+    juan_diego_cuauhtlatoatzin: 'Αγίου Ιωάννη Diego Διδάχου Cuahtlatoatzin',
+    justin_martyr: 'Αγίου Ιουστίνου, μάρτυρος',
+    lawrence_justinian: 'Αγίου Λαυρεντίου Giustiniani',
+    lawrence_of_brindisi_priest: 'Αγίου Λαυρεντίου του Βρινδησίου, παρθένου και διδασκάλου της Εκκλησίας',
+    lawrence_of_rome_deacon: 'Αγίου Λαυρεντίου, διακόνου και μάρτυρος',
+    lawrence_ruiz_and_companions_martyrs: 'Αγίου Λαυρεντίου Ruiz και συντρόφων μαρτύρων',
+    leo_i_the_great_pope: 'Αγίου Λέωντα του Μεγάλου, πάπα και διδασκόλου της Εκκλησίας',
+    louis_grignion_de_montfort_priest: 'Αγίου Λουδοβίκου Μαρία Grignion de Montfort, πρεσβυτέρου',
+    louis_ix_of_france: 'Αγίου Λουδοβίκου',
+    louise_de_marillac_religious: 'Αγίας Λουίζας de Marillac',
+    luke_evangelist: 'Αγίου Λούκα, Ευαγγελιστή',
+    lydia_of_philippi: 'Αγίας Λυδίας της Φιλιππησίας',
+    lydia_of_philippi_first_european_christian: 'Αγίας Λυδίας της Φιλιππησίας, πρώτης Χριστιανής της Ευρώπης', // Custom title
+    mammes_of_caesarea_martyr: 'Αγίου Μάμα, μάρτυρος',
+    marcellin_champagnat_priest: 'Αγίου Μαρκελλίνου Champagnat, πρεσβυτέρου',
+    marcellinus_of_ancona_bishop: 'Αγίου Μαρκελλίνου, επισκόπου',
+    marcellinus_of_rome_and_peter_the_exorcist_martyrs: 'Αγίου Μαρκελλίνου και Πέτρου, μαρτύρων',
+    margaret_mary_alacoque_virgin: 'Αγίας Μαργαρίτας Μαρίας Alacoque, παρθένου',
+    margaret_of_antioch_virgin: 'Αγίας Μαρίνας, παρθενομάρτυρος',
+    margaret_of_scotland: 'Αγίας Μαργαρίτας της Σκοτίας',
+    maria_goretti_virgin: 'Αγίας Μαρίας Goretti, παρθενομάρτυρος',
+    mark_evangelist: 'Αγίου Μάρκου, Ευαγγελιστή',
+    martha_of_bethany: 'Αγίας Μάρθας',
+    martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany: 'Αγίων Μάρθας, Μαρίας και Λαζάρου',
+    martin_de_porres_religious: 'Αγίου Μαρτίνου de Porres, μοναχού',
+    martin_i_pope: 'Αγίου Μαρτίνου Αʹ, πάπα και μάρτυρος',
+    martin_of_tours_bishop: 'Αγίου Μαρτίνου, επισκόπου',
+    mary_magdalene: 'Αγίας Μαρίας της Μαγδαληνής',
+    mary_magdalene_de_pazzi_virgin: 'Αγίας Μαρίας Μαγδαληνής de’ Pazzi, παρθένου',
+    mary_mother_of_god: 'Tης Υπεραγίας Θεοτόκου Μαρίας',
```

Comment:
You are missing the translation of `The Octave Day of the Nativity of the Lord` to Greek.

Note that I am not sure if they use colon (`:`) in Greek, as I haven’t found that character on Greek keyboard layout. Do change it if you find a better punctuation mark.

Also note that `'Tης' !== 'Της'`, as the former uses `T` from Latin alphabet and the latter `Τ` from Greek alphabet.

```suggestion
    mary_mother_of_god: 'Ογδοάδα των Χριστουγέννων: Της Υπεραγίας Θεοτόκου Μαρίας',
```